pkgs/tools/misc/piston-cli/default.nix

```diff
@@ -9,7 +9,7 @@ python3Packages.buildPythonApplication rec {
     sha256 = "qvDGVJcaMXUajdUQWl4W1dost8k0PsS9XX/o8uQrtfY=";
   };
 
-  propagatedBuildInputs = with python3Packages; [ rich prompt_toolkit requests pygments pyyaml ];
+  propagatedBuildInputs = with python3Packages; [ rich prompt_toolkit requests pygments pyyaml more-itertools ];
```

Comment:
Why is this needed suddenly? Is this a regression from a change?

Reply:
I believe it is, I just fails to build now. I'm not sure why. I'm not sure at what point the build starts to fail, as I just noticed it the other day.